### ISSUE TITLE
First "tip" box is cutoff at a weird point

### ISSUE THREAD

**Issue by julianstanley** [datacarpentry/genomics-r-intro#190]
Here's the Tip box:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/22749289/214350133-3bb67d09-713f-488d-88ec-90c394fa5996.png">

Corresponding to this line:

https://github.com/datacarpentry/genomics-r-intro/blob/d34640c1662be74059e932e644428b0297deda16/_episodes_rmd/00-introduction.Rmd#L112

Maybe the author originally wanted it all to be on the header line, but it didn't fit?

How about this instead:

```
> ## Tip: Make sure there are no spaces before or after your URL.
> If your URL has spaces, your web browser may interpret it as a search query.
```

Small issue, is this okay to go ahead and PR?

**Comment by naupaka** [comment]
@julianstanley yes please, a PR would be great. Nice catch!

**Comment by naupaka** [comment]
Closed via #192
